common/djangoapps/course_modes/tests/test_models.py

```diff
@@ -471,3 +471,22 @@ def test_verified_min_price(self, mode_slug, is_error_expected):
             self.assertTrue(is_error_expected, "Did not expect a ValidationError to be thrown.")
         else:
             self.assertFalse(is_error_expected, "Expected a ValidationError to be thrown.")
+
+
+class EnrollmentModeTest(TestCase):
+    """
+    Tests for EnrollmentMode Model.
+    """
+    def tearDown(self):
+        super(EnrollmentModeTest, self).tearDown()
+        EnrollmentMode.objects.all().delete()
```

Comment:
@cahrens here's the best link I could find that explains [default Django testing behavior](https://docs.djangoproject.com/en/1.8/topics/testing/advanced/#using-different-testing-frameworks)

This behavior involves:
1. Performing global pre-test setup.
2. Looking for tests in any file below the current directory whose name matches the pattern test*.py.
3. Creating the test databases.
4. Running migrate to install models and initial data into the test databases.
5. Running the tests that were found.
6. Destroying the test databases.
7. Performing global post-test teardown.